org.winehq.Wine.yml

```diff
@@ -8,7 +8,7 @@ x-gl-version: &gl-version '1.4'
 x-gl-versions: &gl-versions 23.08;1.4
 finish-args:
   - --share=ipc
-  - --socket=x11
+  - --socket=fallback-x11
```

Comment:
this isn't right yet as wayland in wine is still opt-in experimental [feature](https://gitlab.winehq.org/wine/wine/-/releases/wine-9.0#wayland-driver). This is only warning in linter so it's safe to leave as is.

Reply:
Ah, good point.